lib/encoding/zstd/zstd_pure.go

```diff
@@ -0,0 +1,70 @@
+// +build !cgo
```

Comment:
The following tests must be added to `lib/encoding/zstd` package:

1. Compress blocks with various sizes using cgo version, then decompress them with pure Go version.
2. Compress blocks with various sizes using pure Go version, then decompress them with cgo version.
3. Concurrent compression and decompression for blocks with various sizes using cgo and pure Go version.